server/models/show.js

```diff
@@ -18,13 +18,15 @@ module.exports = (sequelize, DataTypes) => {
   };
   Show.init({
     name: DataTypes.STRING,
```

Comment:
Did you add a migration for this? Will be needed to update the DB. Also, I tried to say this yesterday to the whole group, I made a number of edits to the models that might affect the schemas but I think all those changes got incorporated.